content/guides/embed/ui-elements/explorer-metadata.md

```diff
@@ -0,0 +1,245 @@
+---
+rank: 4
+related_endpoints: []
+related_guides:
+  - embed/ui-elements
+required_guides:
+  - embed/ui-elements/installation
+related_resources: []
+alias_paths:
+  - /docs/box-content-explorer
+  - /docs/content-explorer
+---
+
+# Content Explorer - metadata view
+
+With Content Explorer you can also display files and folders
+based on their metadata.
+The metadata view uses [metadata template][template] and [metadata query][metadata-query] to
+find the data you want to display.
+
+## Prerequisites
+
+* Read the [Content Explorer][explorer] guide.
+* Check the [metadata terminology][terminology].
+* Check the information on [metadata queries][metadata-query].
+
+## Create and configure an app
+
+1. [Create a Box app][box-app].
+2. Add the local development address in the CORS Domains: ![CORS Domains](./images/box-app-cors.png)
+3. Generate a [developer token][token].
+
+## Create a metadata template
+
+The next step is to create a metadata template you will use to populate the Content Explorer.
+
+1. Create a metadata template. You can use [Metadata API][creating-templates-api] or [Admin Console][creating-templates-ui] to do so.
+2. Apply an already created template to a Box folder. Make sure you enable the
+cascade policy. For detailed instructions, see
+[instructions on customizing and applying templates][apply-templates].
+
+### Display name and key parameters
+
+* The `displayName` parameter is the display name of the template visible
+in the Admin Console.
+* The `templateKey` parameter is a unique identifier of the template. It needs 
+to be unique across the enterprise for which you create the metadata template.
+If you don't provide the `templateKey` parameter, API creates a unique one
+based on the value in `displayName`.
+* The `[fields].displayName` parameter is the display name of the field as it
+is shown to the user in the web and mobile apps.
+* The `[fields].key` parameter is a unique identifier for a specific field in
+the template. The identifier must be unique within the template to which it
+belongs.
+
+## Display metadata view
+
+To make things easier, you can use a [sample project][metadata-project] to
```

Comment:
Now you can proceed to filling the necessary properties passed to the Content Explorer.

Reply:
To make things easier, you can use a [sample project][metadata-project] based on a simple React app.